app/views/layouts/_navbar.haml

```diff
@@ -4,10 +4,10 @@
       %a.navbar-brand{ href: root_url }= image_tag 'favicon.svg'
     - if @current_user.present?
       - if @current_user.staff?
-        = link_to new_user_url do
-          %button.btn.btn-default.navbar-btn.navbar-left New Driver
         = link_to users_url do
           %button.btn.btn-default.navbar-btn.navbar-left Manage Drivers
+        = link_to import_users_url do
```

Comment:
Here's the whole navbar:
<img width="1280" alt="screen shot 2017-07-03 at 6 04 44 pm" src="https://user-images.githubusercontent.com/3988134/27809015-24adeeea-601a-11e7-81ac-8fc3c77a593f.png">